fmc/src/main.rs

```diff
@@ -35,12 +36,13 @@ Running Caliptra FMC ...
 pub extern "C" fn entry_point() -> ! {
     cprintln!("{}", BANNER);
 
-    if let Some(hand_off) = HandOff::from_previous() {
+    if let Some(mut hand_off) = HandOff::from_previous() {
         let env = fmc_env::FmcEnv::default();
-        hand_off.to_rt(&env)
-    } else {
-        caliptra_drivers::ExitCtrl::exit(0xff)
+        if flow::run(&env, &mut hand_off).is_ok() {
+            hand_off.to_rt(&env)
+        }
```

Comment:
Don't need to decide in this PR, but if something fails during boot we probably want to do one of

1. Try again to boot from rom (maybe invoke an impactless reset?)
2. Notify the SoC and then wait for a new firmware image

This is probably something to discuss with the group.

Reply:
I will file an issue.